### ISSUE TITLE
음식판단 학습 모델 생성

### ISSUE THREAD

**Issue by whdigrace** [ara42/Blog_Project#6]
SOL 1) AI 허브의 음식데이터셋을 학습
-> error : 데이터가 150000개로 너무 방대해서 불러오는데 한계 발생  

SOL 2) Tensorflow 자체의 이미지 학습 model을 이용
- > error : 서비스가 중지됨

SOL 3) AI허브의 음식데이터셋의 일부만을 가져와서 학습 
1. AI 음식데이터셋의 일부를 종류별로 가져와서 food로 라벨링.
  (https://aihub.or.kr/aihubdata/data/view.do?currMenu=115&topMenu=100&aihubDataSe=realm&dataSetSn=79)
2. 공공데이터 포털의 '전라북도 전주시_음식점 사진(가게,메뉴)' 데이터를 가져와서 nonfood로 라벨링
  (https://www.data.go.kr/data/15076758/fileData.do)
3. 기타 인물, 풍경 등의 사진은 개인 데이터를 가져와서 nonfood로 라벨링
4. 총 1000개( food 2340개 nonfood 500개)의 사진 학습
 - Tensorflow.Keras로 학습 
5.overfitting으로 인해 학습이 잘 되지 않음
-> EarlyStopping을  import해서 loss가 최저가 될 때 학습을 중단해서 ovefitting 방지
6. 모델 구축 성공 (  결과 : loss: 0.1149 - accuracy: 0.9590 - val_loss: 0.1487 - val_accuracy: 0.94530)
